spec/models/student_spec.rb

```diff
@@ -1,5 +1,15 @@
 require 'rails_helper'
 
-RSpec.describe Student, :type => :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+RSpec.describe Student, type: :model do
+
+  it "is valid with all attributes" do
+    stu = create(:student)
```

Comment:
avoid shorthand variable naming - can get out of hand quickly and in this case doesn't save many keystrokes

```suggestion
    student = create(:student)
```

Reply:
Also, consider moving the `student` declaration to a [let binding](https://relishapp.com/rspec/rspec-core/docs/helper-methods/let-and-let) and make use of contexts to create different scenarios. (see https://relishapp.com/rspec/rspec-core/v/3-8/docs/example-groups/basic-structure-describe-it).

In this situation there might only be one context needed, but explicitly calling it out allows us to easily see what scenario we are testing here.

```ruby
context 'with valid attributes' do 
  let(:student) { build(:student) }

  it 'is valid' do
    expect(student).to be_valid
  end
end
```

Also see in the above example how I used the `build` strategy instead of `create` ([see here](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#build-strategies) for a list of build strategies in factorybot). This is needed since the `create` strategy will actually save the student to the database and will actually fail if the model is not valid, so your expectation below wouldn't actually mean anything 🙂 

Finally, I like to wrap the thing we are testing in a [subject](https://relishapp.com/rspec/rspec-core/v/3-10/docs/subject/explicit-subject) block. 

Something like the following wraps the student we created with `build(:student)` in a subject block:

```ruby
subject { student }
```

After that, we just need to make sure the subject is valid by changing our expectation block to

```ruby
it 'is valid' do
  expect(subject).to be_valid
```

It might look the same, but at least we can clearly see what we are testing.